exo-setup/features/setup.feature

```diff
@@ -20,14 +20,76 @@ Feature: Setup of Exosphere applications
       | dashboard/Dockerfile     |
       | mongo-service/Dockerfile |
       | web-server/Dockerfile    |
+    And it has generated the file "tmp/docker-compose.yml" with the content:
```

Comment:
This file is not getting checked into Git, so it isn't relevant to the user. It is a technical implementation detail. Let's remove it from this spec.